py/server/test_helper/__init__.py

```diff
@@ -69,11 +69,18 @@ def start_jvm_for_tests(jvm_props: Dict[str, str] = None):
         # Set up a Deephaven Python session
         py_scope_jpy = jpy.get_type("io.deephaven.engine.util.PythonScopeJpyImpl").ofMainGlobals()
         global py_dh_session
+
+        no_op_thread_factory = jpy.get_type("io.deephaven.util.thread.ThreadInitializationFactory").NO_OP
```

Comment:
I think this should follow the _ convention.

Reply:
underscore is for top-level stuff that shouldn't be imported from other modules, not necessary here

Reply:
underscore doesn't apply to auto-variables in the function scope, so the naming is fine here, though I prefer to use the `_j_ ` prefix if an object/variable comes from Java..